src/main/webapp/scripts/group-script.js

```diff
@@ -0,0 +1,63 @@
+var params = JSON.parse(localStorage.getItem('oauth2-test-params'));
+var token = params['access_token'];
+
+function getAllGroups(){
+    // access token expires in 3600 sec after login; fix later
+    console.log(token);
+    fetch('https://www.googleapis.com/admin/directory/v1/groups?domain=groot-test.1bot2.info', {
```

Comment:
The documentation says to use ?domain=groot-test.1bot2.info&customer=my_customer 

Even though this works, it probably is worth doing it the documented way so if something changes to make the API work as documented, it doesn't break.

https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups#get_all_domain_groups